src/browser/pages/vscode.html

```diff
@@ -39,8 +39,8 @@
   <body aria-label=""></body>
 
   <!-- Startup (do not modify order of script tags!) -->
-  <script data-cfasync="false" src="{{CS_STATIC_BASE}}/dist/pages/vscode.js"></script>
-  <script data-cfasync="false" src="{{CS_STATIC_BASE}}/dist/register.js"></script>
+  <script data-cfasync="false" src="{{CS_STATIC_BASE}}/out/browser/register.browserified.js"></script>
```

Comment:
This is completely just a thought I had and not something we need to do in this PR but I wonder if we should do `import "../../register"` in `vscode.ts` and `login.ts` to avoid having to add a second script link (and to keep things to a single bundle especially since I think they both import the logger so we end up bundling it twice).

Reply:
Good call!